README.md

```diff
@@ -53,38 +48,147 @@ See the [Docker documentation](./docs/Docker.md) to get up and running
+  ```
+
+#### Viewing email messages
+
+  To view email messages, Mailcatcher must be running. You can check if it's
+  running by visiting http://localhost:1080/. To run Mailcatcher:
+
+  ```
+  $ mailcatcher
+  ```
+
+  If you would like to run the application on a different port:
+
+  * Change the port number for `mailer_domain_name` and `domain_name` in `config/application.yml`
+  * Run the app on your desired port like `make run PORT=1234`
+
+#### Translations
+
+  Login.gov translates the IdP into English, French and Spanish. To help us handle extra newlines and make sure we wrap lines consistently, we have a script called `./scripts/normalize-yaml` that helps format YAML consistently. After importing translations (or making changes to the *.yml files with strings, run this for the IdP app:
+
+  ```
+  $ make normalize_yaml
+  ```
+
+  [mac-test-passphrase-prompt]: mac-test-passphrase-prompt.png "Mac Test Passphrase Prompt"
+
+  If you would like to preview the translations on a particular page, add
+  `?locale=es` to the end of the URL, such as `http://localhost:3000/?locale=es`.
+  Currently, you'll need to add `?locale=es` to each URL manually.
+
+#### Viewing outbound SMS messages and phone calls
+
+  To see outbound SMS messages and phone calls, visit `http://localhost:3000/test/telephony`.
+
+#### Adding a custom proofing vendor
+
+    We've added the ability for you to use a custom proofing vendor when developing locally. Here is an example of how you would configure that in your local development environment:
+
+    Check out the private repository for `somevendorname`
+
+    ```
+    $ cd vendor
+    $ git clone git@github.com:18F/identity-somevendorname-api-client-gem.git somevendorname
+    ```
+
+    Add appropriate vendor environment variables to `config/application.yml` -- see a member of the
+    login.gov team for credentials and other values.
+
+#### Developing without the internet
 
-If you want to develop without an internet connection, you can set
-`RAILS_OFFLINE=1` in your environment. This disables the `mx` record
-check on email addresses.
+  If you want to develop without an internet connection, you can set
+  `RAILS_OFFLINE=1` in your environment. This disables the `mx` record
+  check on email addresses.
 
-If you want to measure the app's performance in development, set the
-`rack_mini_profiler` option to `'on'` in `config/application.yml` and
-restart the server. See the [rack_mini_profiler] gem for more details.
+  If you want to measure the app's performance in development, set the
+  `rack_mini_profiler` option to `'on'` in `config/application.yml` and
+  restart the server. See the [rack_mini_profiler] gem for more details.
 
-[Laptop]: https://github.com/18F/laptop
-[rack_mini_profiler]: https://github.com/MiniProfiler/rack-mini-profiler
+  [rack_mini_profiler]: https://github.com/MiniProfiler/rack-mini-profiler
 
-#### Testing Analytics
+#### Viewing event and log data
```

Comment:
Maybe not in this PR, but I don't think ANYONE uses this...

Reply:
Removing 💥